### PR TITLE
KAN-25-Create-CRUD-for-Toys

### PR DESCRIPTION
**Pull request by TheTomre**

### CODE REVIEW

File: backend/src/models/Toy.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import mongoose, { Document, Schema } from "mongoose";
+
+const ToySchema: Schema = new Schema({
+  category: { required: true, type: [String] },
```
Comment (MaleryValery) [inline backend/src/models/Toy.ts:4]: category should be unique

File: backend/src/models/Toy.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    default: "available",
+    enum: ["available", "exchanged"],
+    type: String
+  },
+  tokenValue: { default: 0, type: Number }
+});
+
+export default mongoose.model<ToyType>("Toy", ToySchema);
+
+export type ToyType = Document & {
```
Comment (MaleryValery) [inline backend/src/models/Toy.ts:19]: probably should be placed in type folder

File: backend/src/routes/toyRoutes.ts [hunk 3]
```diff
@@ -0,0 +1,102 @@
+/* eslint-disable @typescript-eslint/no-misused-promises */
+import Toy, { ToyType } from "../models/Toy";
+import express, { NextFunction, Request, Response } from "express";
+
+const router = express.Router();
+
+const STATUS = {
```
Comment (MaleryValery) [inline backend/src/routes/toyRoutes.ts:7]: nice, will take it for users
but maybe we should place it in consts folder, so we can use import for every resource

File: backend/src/models/Toy.ts [hunk 4]
```diff
@@ -0,0 +1,27 @@
+import mongoose, { Document, Schema } from "mongoose";
+
+const ToySchema: Schema = new Schema({
+  category: { required: true, type: [String] },
+  description: { required: true, type: String },
+  images: { default: [], type: [String] },
+  name: { required: true, type: String },
+  price: { required: true, type: Number },
```
Comment (MaleryValery) [inline backend/src/models/Toy.ts:8]: price more or equal 0

File: backend/src/routes/toyRoutes.ts [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+  BAD_REQUEST: 400,
+  CREATED: 201,
+  INTERNAL_SERVER_ERROR: 500,
+  NOT_FOUND: 404,
+  OK: 200
+};
+
+// Create Toy
```
Comment (MaleryValery) [inline backend/src/routes/toyRoutes.ts:15]: discussed in telegram, lets move callbacks to controller and service folders

**Review by MaleryValery — COMMENTED** [review]
discussed in Telegram and i have left comments

File: backend/src/models/Toy.ts (hunk 1 above)
Comment (TheTomre) [reply backend/src/models/Toy.ts:4]: Why?

**Pull request merged**